packages/terra-content-container/src/ContentContainer.jsx

```diff
@@ -31,19 +33,19 @@ const ContentContainer = ({
   fill,
   ...customProps
   }) => {
-  const contentLayoutClassNames = classNames([
-    'terra-ContentContainer',
-    { 'terra-ContentContainer--fill': fill },
+  const contentLayoutClassNames = cx([
+    'content-container',
+    { 'content-container-fill': fill },
     customProps.className,
   ]);
 
   return (
     <div {...customProps} className={contentLayoutClassNames}>
-      <div className="terra-ContentContainer-header">
+      <div className={cx('content-container-header')}>
```

Comment:
Based on the css module compilation, this can be shortened to 'header' because it will be namespaced to ContentContainer.